src/scrape_up/codeforces/user.py

```diff
@@ -7,7 +7,7 @@ class USER:
     Fetches user data from CodeForces Competitive Platform
     """
 
-    user_url = "https://codeforces.com/profile/{}"
+    api_base_url = "https://codeforces.com/api/"
```

Comment:
Do not use APIs @lazyMonk1010 |
We are building an API alternative, so it doesn't make any sense to use API under the hood.
Check if you can fix this module using any scraping library(not selenium or chrome web driver).